lib/frrlua.h

```diff
@@ -151,6 +151,21 @@ void lua_decode_integerp(lua_State *L, int idx, long long *num);
  */
 void *lua_tointegerp(lua_State *L, int idx);
 
+/*
+ * Converts a long long to a Lua value and pushes it on the stack.
+ */
+void lua_pushlonglongp(lua_State *L, const long long *num);
+
+void lua_decode_longlongp(lua_State *L, int idx, long long *num);
+
+/*
+ * Converts the Lua value at idx to an int.
```

Comment:
to a *long long

Reply:
updated https://github.com/FRRouting/frr/pull/9012/commits/3bdd49d47495ccd1f58d84d893b8a6de53dd604d